### DISCUSSION TITLE
Where's the list of prefixes for `select`?

### DISCUSSION THREAD

**Discussion by shaperilio** [astral-sh/ruff#14344] [Q&A] [answered]
I can only find snippets here and there, like "I" is for `isort`. Is there a complete list somewhere?

I was confused at first because I was adding stuff to `too.ruff.lint.isort` and hadn't realized I didn't have `"I"` in my `select` so it wasn't actually running.

**Comment by MichaReiser** [comment, marked as answer]
You can find a list of all rules in our [documentation](https://docs.astral.sh/ruff/rules). Each rule code consists of a prefix and a number. E.g. `[unused-import](https://docs.astral.sh/ruff/rules/unused-import/)` is `F401` and you can use `F` as the prefix for all pyflakes rules. You can also use `F4` to select all pyflakes rules that start with `F4`, e.g `F401`, `F402`, ...

**Reply by shaperilio** [reply]
Wow, can't believe I missed that. Thanks.
